<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trie_dechets</class>
 <widget class="QWidget" name="Trie_dechets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="yellowBin_picture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>160</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/bin/picture_yellowBin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="greenBin_picture">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>160</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/bin/picture_greenBin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="blueBin_picture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>160</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/bin/picture_blueBin.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="brownBin_picture">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>160</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/bin/picture_brownBin.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="compost_picture">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>160</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/bin/picture_compost.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="yellowBin_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ici</string>
   </property>
  </widget>
  <widget class="QPushButton" name="greenBin_pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Chez moi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blueBin_pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Juste là</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brownBin_pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Là-dedans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compost_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Celle-ci</string>
   </property>
  </widget>
  <widget class="QLabel" name="question">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Dans quelle poubelle faut-il jeter ce déchet ?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="current_obj">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/waste/0.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="result_message">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_bravo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Image suivante</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
